### PR TITLE
#41 [part 5] : Added accelerometer & gyroscope

### PR DESCRIPTION
**Pull request by OlegNovosad**

### CODE REVIEW

File: lib/src/flutter/events/accelerometer/accelerometer_failed_event.dart [hunk 1]
```diff
@@ -0,0 +1,12 @@
+part of '../../../../telegram_web_app.dart';
+
+/// **Bot API 8.0+**
+/// 
+/// Occurs if a request to start accelerometer tracking fails.
+/// [eventHandler] receives an object with the single field error, describing the reason for the failure. Possible values for error are:
+/// 
+/// **UNSUPPORTED** – Accelerometer tracking is not supported on this device or platform.
+class AccelerometerFailedEvent extends TelegramEvent {
+  AccelerometerFailedEvent(void Function(String error) eventHandler)
```
Comment (khamidjon-khamidov) [inline lib/src/flutter/events/accelerometer/accelerometer_failed_event.dart:10]: @OlegNovosad I think you should wrap the error inside an object according to the api definition.

File: lib/src/flutter/events/gyroscope/gyroscope_failed_event.dart [hunk 2]
```diff
@@ -0,0 +1,12 @@
+part of '../../../../telegram_web_app.dart';
+
+/// **Bot API 8.0+**
+/// 
+/// Occurs if a request to start gyroscope tracking fails.
+/// eventHandler receives an object with the single field error, describing the reason for the failure. Possible values for error are:
+/// 
+/// **UNSUPPORTED** – Gyroscope tracking is not supported on this device or platform.
+class GyroscopeFailedEvent extends TelegramEvent {
+  GyroscopeFailedEvent(void Function(String error) eventHandler)
```
Comment (khamidjon-khamidov) [inline lib/src/flutter/events/gyroscope/gyroscope_failed_event.dart:10]: @OlegNovosad I think you should wrap the error inside an object according to the api definition.

File: lib/src/flutter/events/accelerometer/accelerometer_failed_event.dart (hunk 1 above)
Comment (OlegNovosad) [reply lib/src/flutter/events/accelerometer/accelerometer_failed_event.dart:10]: According to the api it is just string error, just the same as we do we device orientation and others.

File: lib/src/flutter/events/gyroscope/gyroscope_failed_event.dart (hunk 2 above)
Comment (OlegNovosad) [reply lib/src/flutter/events/gyroscope/gyroscope_failed_event.dart:10]: According to the api it is just string error, just the same as we do we device orientation and others.

Comment (OlegNovosad) [reply lib/src/flutter/events/gyroscope/gyroscope_failed_event.dart:10]: Ok. Will also fix it for other failed events (device orientation, home screen etc.)

Comment (OlegNovosad) [reply lib/src/flutter/events/gyroscope/gyroscope_failed_event.dart:10]: Fixed.

**Pull request merged**